simple/stats/db.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sqlite3
+
+from stats.data import Observation
+from stats.data import Triple
+
+_CREATE_TRIPLES_TABLE = """
+create table if not exists triples (
+    subject_id TEXT,
+    predicate TEXT,
+    object_id TEXT,
+    object_value TEXT
+);
+"""
+
+_INSERT_TRIPLES_STATEMENT = "insert into triples values(?, ?, ?, ?)"
+
+_CREATE_OBSERVATIONS_TABLE = """
+create table if not exists observations (
+    entity TEXT,
+    variable TEXT,
+    date TEXT,
+    value TEXT,
+    provenance TEXT
+);
+"""
+
+_INSERT_OBSERVATIONS_STATEMENT = "insert into observations values(?, ?, ?, ?, ?)"
+
+
+class Db:
```

Comment:
call this SqliteDB since this is specific for sqlite and you will need one for cloudsql

Reply:
Also could this be general? In https://github.com/datacommonsorg/mixer/blob/a03c205f7783c416255a3fdf03af0cd87f29cc38/internal/sqldb/writer.go#L114-L146, they functions would work for both sqlite and cloudsql

Reply:
> call this SqliteDB since this is specific for sqlite and you will need one for cloudsql

I intentionally did not call it that. We'll keep the same `Db` interface when we add support for other dbs and make the implementation adapt to other dbs.

This keeps other parts of the importer agnostic of the actual db.

Reply:
> functions would work for both sqlite and cloudsql

`self.db.executemany` achieves that generalization (similar to the `sqlClient.Exec` call [here][exec]). We could have a helper that takes an insert statement and tuples but the for-comprehension is concise enough so didn't feel a need for that helper.

[exec]: https://github.com/datacommonsorg/mixer/blob/a03c205f7783c416255a3fdf03af0cd87f29cc38/internal/sqldb/writer.go#L89

Reply:
That's good then, the current PR covers both points!